src/dewey/events.clj

```diff
@@ -0,0 +1,26 @@
+(ns dewey.events
+  (:require [clojure.tools.logging :as log]
+            [langohr.basic :as lb]
+            [dewey.config :as cfg])
+  (:import [org.cyverse.events.ping PingMessages
+                                    PingMessages$Ping
+                                    PingMessages$Pong]
+           [com.google.protobuf.util JsonFormat]))
+
+(defn- ping-handler
```

Comment:
This handler is nearly identical to the one in `info-typer` at some point in the future, we should consolidate this functionality. For now, it's fine.